keanu-python/tests/test_vertex.py

```diff
@@ -13,6 +15,24 @@ def jvm_view():
     return jvm_view
 
 
+def assert_vertex_value_equals_scalar(vertex, vertex_type, scalar):
```

Comment:
Functions within `test_cast.py` are all annotated. Should these also be annotated so that it's consistent?

Reply:
There is another task to finish up the annotations for all tests! My first pr just covered a few

Reply:
Ah, sounds good!